index.bs

```diff
@@ -5294,38 +5294,40 @@ partial interface MLGraphBuilder {
     // of the input values itself, in order to increase the numerical stability of
     // the result.
     // [1]: https://cs231n.github.io/linear-classify/#softmax
-    const max_x = builder.reduceMax(x, { axes: [1], keepDimensions: true });
+    const max_x = builder.reduceMax(x, { axes: [axis], keepDimensions: true });
     const exp_x = builder.exp(builder.sub(x, max_x));
-    return builder.div(exp_x, builder.reduceSum(exp_x, { axes: [1], keepDimensions: true }));
+    return builder.div(exp_x, builder.reduceSum(exp_x, { axes: [axis], keepDimensions: true }));
```

Comment:
```suggestion
    const maxX = builder.reduceMax(x, { axes: [axis], keepDimensions: true });
    const expX = builder.exp(builder.sub(x, maxX));
    return builder.div(expX, builder.reduceSum(expX, { axes: [axis], keepDimensions: true }));
```
I don't know why just this place uses `snake_case` whereas the others use camelCase.

Reply:
Done in 7f33accb64e89ba4c9ce41fce14ee0b7b120304a